app-editors/featherpad/featherpad-1.0.0.ebuild

```
@@ -0,0 +1,38 @@
+# Copyright 2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit xdg cmake
+
+DESCRIPTION="Lightweight Qt5 Plain-Text Editor for Linux"
+HOMEPAGE="https://github.com/tsujan/FeatherPad"
+SRC_URI="https://github.com/tsujan/FeatherPad/archive/V${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="+X"
+
+RDEPEND="app-text/hunspell
```

```suggestion
RDEPEND="app-text/hunspell:=
```

`app-text/hunspell` has a subslot, meaning we want to auto rebuild `featherpad` when the subslot changes.
For more info, you can read the [devmanual](https://devmanual.gentoo.org/general-concepts/slotting/index.html#sub-slots).